packages/core/src/components/overlay/overlay.tsx

```diff
@@ -107,6 +107,9 @@ export interface IOverlayProps extends IOverlayableProps, IBackdropProps, IProps
      */
     isOpen: boolean;
 
+    /** CSS class name to add to the detached container root element (if not `inline`). */
```

Comment:
This is a little confusing: the `portalClassName` is added whether or not the content is inline, yes? Should we state instead that this prop is particularly useful when the content is not inline?